cli/initiator/reshare.go

```diff
@@ -0,0 +1,231 @@
+package initiator
+
+import (
+	"crypto/rsa"
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv-dkg/cli/flags"
+	cli_utils "github.com/bloxapp/ssv-dkg/cli/utils"
+	"github.com/bloxapp/ssv-dkg/pkgs/crypto"
+	"github.com/bloxapp/ssv-dkg/pkgs/initiator"
+	"github.com/bloxapp/ssv-dkg/pkgs/utils"
+)
+
+func init() {
+	flags.InitiatorPrivateKeyFlag(StartReshare)
+	flags.InitiatorPrivateKeyPassFlag(StartReshare)
+	flags.WithdrawAddressFlag(StartReshare)
+	flags.OperatorsInfoFlag(StartReshare)
+	flags.OperatorIDsFlag(StartReshare)
+	flags.OldIDFlag(StartReshare)
+	flags.NewOperatorIDsFlag(StartReshare)
+	flags.OwnerAddressFlag(StartReshare)
+	flags.NonceFlag(StartReshare)
+	flags.ConfigPathFlag(StartReshare)
+	flags.LogLevelFlag(StartReshare)
+	flags.LogFormatFlag(StartReshare)
+	flags.LogLevelFormatFlag(StartReshare)
+	flags.LogFilePathFlag(StartReshare)
+	flags.ResultPathFlag(StartReshare)
+	if err := viper.BindPFlag("withdrawAddress", StartReshare.PersistentFlags().Lookup("withdrawAddress")); err != nil {
```

Comment:
also here I think it's best to move this to a function as since its repeated a lot. also we're using the raw strings for the flag names and they are defined as consts in `flags`, better use the consts so we don't have to change many times if something has changed.

Reply:
honestly this is more suitable to happen in #20 , but these PRs and kinda overlapping now